docs/Actions/send-email.md

```diff
@@ -0,0 +1,564 @@
+| Attach from Token             | Computes file path from token. It must be relative to the portal home folder. Multiple items must be separated by semicolons.                                                                                                                                             | Yes             | `empty string`                           |
+| Ignore Errors                 | If errors are ignored, the submission of the form continues even when there is an error sending the email. Otherwise, an error message will be displayed to the user. Note that if the email is being sent async or is placed in a pickup folder, errors won't be caught. | No              | No                                       |
+
+## `Formats`
+
+| Format                                   | Details                                                                                                                                                                                                                      |
+| ---------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Html (And replace newline with `<br />`) | To simplify the input of the email we are replacing the newlines in the `Body` of the email with html tag `<br>`. While it is easier to compose the email it may have strange behaviors with some email clients like Outlook |
+| Text                                     | Regular text, with no formatting options such as bold, italics, underlines, or special layout. Can include URLs, such as http:// and ftp:// options.                                                                         |
+| Html (**Recommended**)                   | It’s the way web pages and email templates are coded so that text is formatted and images are added . What it is displayed in the HTML editor of `Body` it is the the result that will be displayed in the email client      |
+
+## `Security`
+
+:::caution
+Note that this presents a security risk when the To, From, CC, BCC and Reply To parameter has Tokens from untrusted sources.
+:::
+
+The above mentioned parameters support tokens so the values, if not from a trusted source, may be altered and emails recipients can be modified. 
+For example if you let the user to input the email address manually from a Form may be a security risk. The expectation is to input only one email address but the end user may input: `my-email@company.com;other-email@othercompany.com`. Doing this he may send the emails that are meant only for him to other people or organizations. To avoid similar situations we strongly recommend using Tokens like `[User:Email]` or hardcoded email addresses when setting recipients parameters. 

```

Comment:
```suggestion
For example, if you let the user input the email address manually from a Form, it may present a security risk. The expectation is to input only one email address but the end-user may input: `my-email@company.com;other-email@othercompany.com`. Doing this he may send the emails that are meant only for him to other people or organizations. To avoid similar situations we strongly recommend using Tokens like `[User:Email]` or hardcoded email addresses when setting recipients parameters. 
```